### PR TITLE
fixed typos in *.proto files

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally using ``cargo fmt`` command prior to submission?
3. [ ] Have you checked your code using ```cargo clippy``` command?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

fixed typo and small grammatical errors in the documentation.

### CODE REVIEW

**Comment by agourlay** [comment]
Thank you for your initiative :+1:  

It seems you are modifying the `docs.md` files manually, however those are generated based on the proto files.

This means that you should first fix the typos in the proto files and then run `./tools/generate_grpc_docs.sh`

**Comment by tusharxoxoxo** [comment]
[
Screenshot 2023-03-21 at 16 23 46
](url)

File: lib/api/src/grpc/proto/points.proto [hunk 1]
```diff
@@ -402,8 +402,8 @@ message GeoBoundingBox {
 }
 
 message GeoRadius {
-  GeoPoint center = 1; // Center of the circle
```
Comment (agourlay) [inline lib/api/src/grpc/proto/points.proto:405]: those two changes are British English, I'd rather stick to the American variants please

**Comment by agourlay** [comment]
One nitipick and then it is good to go, sorry for the slow turnaround on this PR.

**Comment by agourlay** [comment]
@tusharxoxoxo I took the liberty to push to your branch to streamline the process.

I fixed the British English comment and various syncing issues.

The flow to fix gRPC doc typo is

1. fix typo in proto file
2. compile to generate rust file
3. generate gRPC docs

**Review by agourlay — APPROVED** [review]

**Pull request merged**